Tools/Postprocessing/C_Src/WritePlotfileToASCII.cpp

```diff
@@ -131,16 +134,20 @@ main (int   argc,
 
         const Array4<Real>& mfdata = mf_onegrid.array(mfi);
 
-       for (auto n=0; n<ncomp; ++n) {
-       for (auto k = lo.z; k <= hi.z; ++k) {
-       for (auto j = lo.y; j <= hi.y; ++j) {
-       for (auto i = lo.x; i <= hi.x; ++i) {
-            Print() << i << " " << j << " " << k << " " << n << " " << mfdata(i,j,k,n) << std::endl;
-       }
-       }
-       }
-       }
-            
+        if (fast == 1){
+          // This way is much faster than the loops below
+          amrex::Print() << mf_onegrid[mfi];
+        }else{
+          for (auto n=0; n<ncomp; ++n) {
+            for (auto k = lo.z; k <= hi.z; ++k) {
+              for (auto j = lo.y; j <= hi.y; ++j) {
+                for (auto i = lo.x; i <= hi.x; ++i) {
+                  Print() << i << " " << j << " " << k << " " << n << " " << mfdata(i,j,k,n) << std::endl;
```

Comment:
@emotheau Could you test to see if the change below makes it as fast as the "fast" option?
```suggestion
                  Print() << i << " " << j << " " << k << " " << n << " " << mfdata(i,j,k,n) << "\n";
```

Reply:
I tested and it does not really help, it is as slow as the `std::endl` version. I think the loop over the n components is making things bad.

Reply:
Sorry, can you test it again?  It should use `std::cout` instead of `Print`.  The later has flush internally.

Reply:
Ah ! This time it is almost as fast as the "fast" option ! Do we keep this suggestion and I remove all my previous stuff ?

Reply:
Maybe you can keep both options, but choose a different name for the argument to show the difference in the order of the printed data.

I also think you should use `std::cout` instead of `Print()` too for the new option you added.

Reply:
Ok I updated my PR to reflect your suggestions. Both methods give different representation of data but with the same (roughly) computational time. I think we are good now?